src/core/components/operation-summary-path.jsx

```diff
@@ -34,17 +28,22 @@ export default class OperationSummaryPath extends PureComponent{
       isDeepLinkingEnabled,
     } = operationProps.toJS()
 
+    // add <wbr> word-break elements between each segment, before the slash
```

Comment:
Just nitpick here. Do we want to provide more descriptive explanation why this is here?
Here is my initial suggetion.

```suggestion
    /**
     * Add <wbr> word-break elements between each segment, before the slash
     * to allow browsers an opportunity to break long paths into sensible segments.
     */
```